frontend/src/HomePage/AppList.jsx

```diff
@@ -2,6 +2,22 @@ import React from 'react';
 import AppCard from './AppCard';
 
 const AppList = (props) => {
+
+  const basestyles = {
```

Comment:
Can we move all CSS to `frontend/src/_styles/theme.scss` and use class names instead?

Reply:
sure , will update